### ISSUE TITLE
[BUG] Script generation is not being outputed into ST UI when using KoboldCpp 1.77

### ISSUE THREAD

**Issue by vorpal-void** [SillyTavern/SillyTavern#3072]
### Environment

🪟 Windows

### System

Windows 11 Pro 10.0.22631

### Version

1.12.7

### Desktop Information

- Node.js version: v20.12.2
- Generation API: KoboldCpp 1.77
- Model: EVA-Qwen2.5-14B-v0.2-Q4_K_M
- Instruct: ChatML, no system prompt

### Describe the problem

For testing purposes, I've done a clean install of ST and created an extremely simple Quick Reply script. 
It works as the console output shows the expected result, but things are not shown in the UI with the following error:

```
Endpoint error: SyntaxError: No number after minus sign in JSON at position 4803
    at JSON.parse (<anonymous>)
    at Response.json (file:///E:/SillyTavern/node_modules/node-fetch/src/body.js:149:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async file:///E:/SillyTavern/src/endpoints/backends/text-completions.js:358:30
```

_Although, everything works perfectly fine with KoboldCpp 1.76._ 

I'm not sure where exactly to report it since I'm not using KoboldCpp without ST and don't know how to reproduce it just with Kobold alone.

<details>
  <summary>Script just in case</summary>  

  ```
  /gen [length=100] (Pause roleplay. Summarize chat history so far.) |
  /popup {{pipe}}
  ```  

</details>

### Please tick the boxes

- [X] I have explained the issue clearly, and I included all relevant info
- [X] I have checked that this [issue hasn't already been raised](https://github.com/SillyTavern/SillyTavern/issues?q=is%3Aissue)
- [X] I have checked the [docs](https://docs.sillytavern.app/) ![important](https://img.shields.io/badge/Important!-F6094E)

**Comment by Cohee1207** [comment]
Try disabling logprobs request.

**Comment by Cohee1207** [comment]
This is confirmed with KoboldCpp developers that some samplers produce invalid JSON when logprobs are requested. This will be fixed in 1.78. Until then, disable logprobs in user settings or stay use 1.76. Closing the bug since it's not an ST issue.
